Engine/Cli/CliActionExecutor.cs

```diff
@@ -269,26 +269,34 @@ string getVersion()
 
                     return TypeData.FromType(typeof(CliActionExecutor)).Assembly.SemanticVersion.ToString(); // OpenTAP is not installed. lets just return this. 
                 }
+                string tapCommand = OperatingSystem.Current == OperatingSystem.Windows ? "tap.exe" : "tap";
 
-                Console.WriteLine("OpenTAP Command Line Interface ({0})", getVersion());
-                Console.WriteLine("Usage: tap <command> [<subcommand(s)>] [<args>]\n");
+                log.Info("OpenTAP Command Line Interface ({0})", getVersion());
+                if (args.Length == 0)
+                {
+                    log.Info($"\"{tapCommand}\" is not a recognized command.");
+                }
+                else
+                {
+                    log.Info($"\"{tapCommand} {string.Join(" ", args)}\" is not a recognized command.");
```

Comment:
Maybe this should be log.Error to highlight the problem?

Reply:
This is now a log.Error instead.